Resources/Locale/en-US/deltav/paper/paper-misc.ftl

```diff
@@ -9,4 +9,13 @@ book-text-lunchbox-healthy = Hello Honey!
 book-text-lunchbox-unhealthy = Hello Honey!
       I've packed you something fun for this shift! I hope you enjoy it!!
       Love you so so much,
-       - Mum.
\ No newline at end of file
+       - Mum.
+book-text-subscription-notice = [head=2]Greetings loyal subscriber,[/head]
+      
+      We hope you enjoy this months edition of Gorlex Girlfriends™ sponsored by the Syndicate.
+      
+      With this months issue we will be heading behind the seens with some of our famous actors working on the amazing "[color=#780606]Nukies[/color]" TV show and maybe show some teasers for next season. Read up on the personal interview with beloved Operative November as she explains the troubles of fake blood getting in her hair!
```

Comment:
scenes not seens

Reply:
Grammarly really didn't help me much with helping these fuckups smh